otter/integration/tests/test_lbsh.py

```diff
@@ -34,6 +38,7 @@ class TestLoadBalancerSelfHealing(unittest.TestCase):
     This class contains test cases to test the load balancer healing
     function of the Otter Converger.
     """
+    timeout = 1800
```

Comment:
Is this used anywhere? It looks to me like everything is using timeout_default, though I could be missing something.